### ISSUE TITLE
Attribute type error

### ISSUE THREAD

**Issue by jeihanraihan** [sodrooome/django-web-polls#2]
when I tried to set up in the login page, suddenly an error occurred, but if it goes directly to the blog page nothing happens. Would you help me out? Thanks before

**Comment by sodrooome** [comment]
yeah, I think to take the vote in current cases for now it can only be done by configuring it to the docs page only. For other pages, there is no further update. Maybe i'll be add to that feature in the meantime.

**Comment by jeihanraihan** [comment]
Alright, that would be appreciated
